app/views/layouts/_select_language_list.html.erb

```diff
@@ -2,17 +2,23 @@
   <% AVAILABLE_LANGUAGES.each do |language| %>
     <li>
       <%= link_to({ :locale => language[:code] },
-                  { :class => ["btn btn-link text-start", { "bg-body-secondary text-secondary-emphasis" => language[:code] == I18n.locale.to_s }],
+                  { :class => ["btn btn-link text-start",
+                               { "bg-body-secondary text-secondary-emphasis" => language[:code] == I18n.locale.to_s }],
                     :data => { :bs_dismiss => "modal",
                                :language_code => language[:code] } }) do %>
-        <% if language[:short_native_name] %>
-          <%= language[:short_native_name] %>
-          <% if language[:short_native_note] %>
-            <small>(<%= language[:short_native_note] %>)</small>
+        <span class="native_name d-block">
+          <% if language[:short_native_name] %>
+            <%= language[:short_native_name] %>
+            <% if language[:short_native_note] %>
+              <small>(<%= language[:short_native_note] %>)</small>
+            <% end %>
+          <% else %>
+            <%= language[:native_name] %>
           <% end %>
-        <% else %>
-          <%= language[:native_name] %>
-        <% end %>
+        </span>
+        <small lang="en" class="english_name d-block text-secondary">
+          <%= language[:english_name] %>
+        </small>
```

Comment:
Are we hard-coding English only because most fonts support it, or are there other reasons? Translated language names would be useful to users who want to see the map in a language other than their own, but in that case, we should try to localize the language name into the current language, falling back to English. I think we can assume that the user has an adequate font for the language they currently prefer.

/ref https://github.com/openstreetmap/openstreetmap-website/pull/5201#discussion_r2086125665

Reply:
> I think we can assume that the user has an adequate font for the language they currently prefer.

Not true as soon as you click on any language that doesn't have a font.

Reply:
Right, each language will be labeled with both the native name and the English name – but not the name in the currently selected language. I think we’re going to get people wondering why there’s bits of English that never change no matter what you set as your language. It’s a very unusual approach.

Reply:
@1ec5 If you propose using `new Intl.DisplayNames([lang], { type: "language" }).of(lang)`, you can open a new issue, then I'll ask you what are your plans for `lang = be-Tarask`, `lang = aln` and other corner cases, and you'll be able to describe your plans in a place more convenient than comments for a merged pull request.

Reply:
Sure, I’m not trying to make a scene, just trying to better understand the rationale for this change. But I opened #6319 to try to articulate an alternative.